MERP/merp_2nd_ed_sheet.html

```diff
@@ -3778,11 +3779,11 @@ <h3>Status</h3>
                                         <span class="skill-label" title="The effective movement maneuver bonus equals MM bonus from Movement Speed minus the total MM penalty">Effective MM Bonus:</span>
                                     </div>
                                     <div class="table-cell">
-                                        <input type="hidden" name="attr_creature_total_mm_penalty" value="0" />
-                                        <input type="number" class="autocalc-total sheet-autocalc-bonus" name="attr_creature_total_mm_display_penalty" value="(@{creature_total_mm_penalty})" disabled/> feet/rnd
+                                        <input type="hidden" name="attr_creature_effective_mm_bonus" value="0" />
+                                        <input type="number" class="autocalc-total sheet-autocalc-bonus" name="attr_creature_effective_mm_display_bonus" value="(@{creature_mm_bonus}+@{creature_total_mm_penalty})" disabled/> feet/rnd
```

Comment:
This seems to be a cut/paste which wasn't updated.  Changed it to show effective MM bonus (bonus+penalty).  Probably shouldn't be feet/rnd, but I can fix that another time.